odl/tomo/backends/stir_bindings.py

```diff
@@ -170,7 +168,7 @@ def adjoint(self):
 
 class BackProjectorByBinWrapper(Operator):
 
-    """A backprojector using STIR."""
+    """A back projector using STIR."""
```

Comment:
Well its Swedish as well and odl is basically a German Swedish joint venture. (Finland is basically [East-Sweden](http://sv.urbandictionary.com/define.php?term=East-Sweden))